product-features/self-service-listing.md

```diff
@@ -132,15 +132,29 @@ Recommended to track transaction status:
 
 </details>
 
-### Step 2: Contract creation
+### Step 2: LP Lock & Burn Settings
+
+After submitting the Self-Service Listing Pool, a pop-up will appear, allowing the creator to lock or burn the LP tokens, or to leave the liquidity pool unlocked. By default, the Self-Service Listing Pool is set to be locked for 6 months, as it is the recommended option. Users are prompted to select one of three settings:
```

Comment:
"... allowing the creator to lock or burn the LP tokens, or to leave the liquidity pool unlocked." podría ser algo así como "allowing the creator **to choose whether to** lock or burn the **initial** LP tokens, or o leave the liquidity pool unlocked."

Unlocked me suena medio repetitivo, por ahí re frasearlo para que se entienda que los LP tokens irian a su wallet y puede hacer withdraw cuando quiera. O algo así.

Son sólo sugerencias nomás.